examples/mqtt/device_comms/iotcc/samplePropMqtt.conf

```diff
@@ -15,6 +15,10 @@ DevicePropList = {"Country":"USA-G", "State":"California", "City":"Palo Alto", "
 WebSocketUrl = "Websocket-address-url"
```

Comment:
Yes, because this example folder handles the connection to DeviceComms using MQTT and connection to DCCComms using Websocket.